### PR TITLE
Add release note for 1.20.1

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: site/src/pages/release-notes/1.20.1.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date: 2022-10-11
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- N/A
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.20.1.mdx:11]: Remove all `N/A`s?

File: site/src/pages/release-notes/1.20.1.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## 🛠️ Bug fixes
+
+- `application/json` media type is correctly set to HTTP/JSON-to-gRPC transcoding responses. #4474
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.20.1.mdx:15]: What do you think of also documenting the affected versions?

```suggestion
- `application/json` media type is correctly set for HTTP/JSON-to-gRPC transcoded responses. #4474
    - Versions 1.19.0 to 1.20.0 are affected
```

**Review by jrhee17 — APPROVED** [review]
Thanks @minwoox ! 🙇 👍 🙇

File: site/src/pages/release-notes/1.20.1.mdx (hunk 2 above)
Comment (ikhoon) [reply site/src/pages/release-notes/1.20.1.mdx:15]: ; and link `#4473` as well?

Comment (minwoox) [reply site/src/pages/release-notes/1.20.1.mdx]: Fixed. 😉

**Review by ikhoon — APPROVED** [review]
Thanks!

**Pull request merged**